src/thor/costmatrix.cc

```diff
@@ -890,5 +929,111 @@ void CostMatrix::SetTargets(baldr::GraphReader& graphreader,
   }
 }
 
+// Form the path from the adjacency list.
+void CostMatrix::RecostPaths(GraphReader& graphreader,
```

Comment:
it would be cool to move the inner loop of recosting a single src and tgt to a common area like path info or somethign else that makes sense so that this and bidir a* can share it

Reply:
left a todo